lib/api/repository/survey_repository.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter_survey/api/survey_service.dart';
+import 'package:flutter_survey/api/exception/network_exceptions.dart';
+import 'package:flutter_survey/model/surveys_model.dart';
+import 'package:injectable/injectable.dart';
+
+abstract class SurveyRepository {
+  Future<SurveysModel> getSurveys(
+      {required int pageNumber, required int pageSize});
```

Comment:
```suggestion
      {required int pageNumber, required int pageSize}
      );
```

Reply:
better to format it like this, by adding 1 more `,` after `pageSize` param + reformatting

```
  Future<SurveysModel> getSurveys({
    required int pageNumber,
    required int pageSize,
  });
```